src/components/common/Builder/FormBuilder/GenerateField.tsx

```diff
@@ -0,0 +1,43 @@
+import { Controller } from "react-hook-form";
+import { Label } from "../../Label";
+import type { FC, ComponentType } from "react";
+import type { Control } from "react-hook-form";
+import type { PropertyMeta } from "../../../../services/hubspot/types";
+
+type Props = {
+    meta: PropertyMeta;
+    Component: ComponentType<Record<string, unknown>>;
+    control: Control<Record<string, unknown>>,
+    value?: unknown;
+};
+
+const GenerateField: FC<Props> = ({ meta, control, Component }) => {
+    const fieldType = meta?.fieldType;
+    const label = meta?.label;
+
+    if (!meta) {
```

Comment:
This will never happen according to the types, this is always a `PropertyMeta` type

Reply:
fixed type

Reply:
I'm not sure that was the correct solution because all typing was correct before which meant every usecase of this was been passed a `PropertyMeta` so there was no need for it to now be marked optional.

Reply:
moved these checks to the parent component.